### DISCUSSION TITLE
Example Change #1: Humanitarian Scope Vocabulary

### DISCUSSION THREAD

**Discussion by robredpath** [IATI/standard-process#4] [Demos and examples]
In our meeting earlier this month, we looked an issue that exists when people try to work with IATI data that uses the Humanitarian Scope Vocabulary in spreadsheets, as an example of the kind of real-world issues that we deal with in standard governance. 

We've copied the example into this repo as an example change request: https://github.com/IATI/standard-process/blob/main/example_changes/humanitarian_scope_vocab.md

It would be useful to hear views from the group on what IATI's response to this issue should be. We can set aside for a moment the existing upgrade process and consider what we think a good response looks like. As we discuss, we may uncover principles that can contribute to the [discussion about principles](https://github.com/IATI/standard-process/discussions/2) that @stevieflow has opened.

**Comment by robredpath** [comment]
In the meeting, responses included:
* Do nothing to the standard, and issue guidance for spreadsheet users around how to deal with this issue. The consequence of this is that users have to do work instead of publishers - but that is a choice that is open to us. 
* Introduce the new codes alongside the old codes, and encourage migration over time. Possibly, we could remove the old codes once e see that they are no longer used. Update IATI Secretariat-maintained tools to only output the new codes to ease data use, and encourage the creators of other tools to do the same. The consequence of this is that there may be more dependence upon IATI Secretariat tools and that it makes it harder for people to use IATI data directly from publishers, but we believe that this is a minority of data use in reality. 

We would welcome any further thoughts on this.

**Reply by bill-anderson** [reply]
Agree 100% with the second bullet.

**Comment by sylvanr** [comment]
It would be interesting to review all Codelist items according to the output of this discussion, and standardise these definitions. This could be a session with volunteers from this stewardship working group.

**Comment by davidmegginson** [comment]
The challenge with bullet 2 is that it offloads the complexity from data producers (saving them from having to transition right away) to data consumers (forcing them to deal with multiple, inconsistent code lists).

Sadly, there's no silver bullet — any solution that saves bother to one group increases bother to the other.
